src/zentables/zentables.py

```diff
@@ -840,53 +832,63 @@ def _combine_n_pct(
     return df_n_str.add(df_pct_str).fillna(f"{suppress_symbol}")
 
 
-def _is_between(x, ceiling: int = 10):
+def _is_between(val, low: int = 1, high: int = 10):
     """
     Helper function that verifies if the input is below ceiling value
     """
-    return x in range(1, ceiling)
+    return low <= val < high
 
 
-def _add_random_noise(x, rng):
-    """
-    Helper function that adds random noise to entry according a Uniform distribution
-    """
-    return x + rng.uniform(0, 1)
+def _seed_to_rng(seed: Optional[Union[int, Generator]] = None) -> Generator:
+    if seed is None:
+        return np.random.default_rng()
+    if isinstance(seed, int):
+        return np.random.default_rng(seed)
 
 
-def _local_suppression(mini_df_n: pd.DataFrame, ceiling: int = 10, seed: int = 2021):
+def _local_suppression(
+    mini_df_n: pd.DataFrame,
+    low: int = 1,
+    high: int = 10,
+    seed: Optional[Union[int, Generator]] = None,
+):
     """
-    Helper function that applies cell suppression to
+    Helper function that applies cell suppression to mini_df_n.
+    mini_df_n is assumed to be a smaller portion of a larger
     """
-    rng = np.random.default_rng(seed)
-    mask = mini_df_n.applymap(lambda entry: _is_between(entry, ceiling))
+    rng = _seed_to_rng(seed)
+    mask = np.logical_and(mini_df_n >= low, mini_df_n < high)
+    tie_beaking_number = mini_df_n.abs().max().max() * 2
```

Comment:
nitpick, but this isn't the tie breaking number; it's the "already zeroed out" number.